lib/oca-epak/epak/client.rb

```diff
@@ -100,22 +177,30 @@ def get_operation_codes
       # @param [String] "From date" in DD-MM-YYYY format
       # @param [String] "To date" in DD-MM-YYYY format
       # @return [Array, nil] Contains an array of hashes with NroProducto and NumeroEnvio
-      def list_shipments(cuit, from_date, to_date)
+      def list_shipments(opts = {})
```

Comment:
The idea was to make the `from_date` and `to_date` optional and instead of make them `nil` by default, I checked all the other methods that were handling an `opts = {}` as an argument, so move to this approach. 

But I understand that this can break other functionality, so let me know if you want to go back to how it was.